beacon/beacon-basic-string.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>W3C Beacon Basic String Test</title>
+    <meta name="timeout" content="long">
+    <meta name="author" title="Microsoft Edge" href="https://www.microsoft.com">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <script src="/common/utils.js"></script>
+    <script src="beacon-common.js?pipe=sub"></script>
+    <script>
+        "use strict";
+        runTests(stringTests);
```

Comment:
The string is large, but not that large.. Do you have any suggestions for how to improve this?